src/components/Encounter.css

```diff
@@ -110,4 +110,41 @@
 
 .delete-button:hover .delete-tooltip {
   visibility: visible;
+}
+
+@media only screen and (max-width: 500px) {
+  .encounter-list {
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+  }
+
+  .encounter-element {
+    width: 90%;
+  }
+
+  .encounter-tag {
+    box-shadow: -1px -1px 3px var(--light),
+      3px 3px 5px var(--dark);
+    font-size: large;
+    height: 25px;
+    width: 90%;
+  }
+
+  .delete-button {
+    border: none;
+    box-shadow: -1px -1px 3px var(--light),
+      3px 3px 5px var(--dark);
+    height: 35px;
+    font-size: medium;
+    left: 25px;
+    margin: .3em;
+    position: relative;
+    width: 50%;
+  }
+
+  .delete-button:hover .delete-tooltip {
+    visibility: hidden;
```

Comment:
I didn't realize tooltips needed to be declared for mobile. Are we going to change the event listener?